rootcmd/root.go

```diff
@@ -80,7 +80,7 @@ func subCommands(config *hazelcast.Config) []*cobra.Command {
 		{Name: "Topic", IssueNum: 53},
 	}
 	for _, fd := range fds {
-		cmds = append(cmds, fakeDoor.MakeFakeCommand(fd))
+		cmds = append(cmds, fakeDoor.NewFakeCommand(fd))
```

Comment:
Was this change necessary?

Reply:
It is not necessary, but it is more go idiomatic way of naming I guess.